### ISSUE TITLE
Platform compatibility tables are extensible in configuration

### ISSUE THREAD

**Issue by serialseb** [OpenWrap/openwrap#11]
Platform compatibility (aka which platform can consume which assemblies and in which order in the /bin exports) is currently fixed. It would be immensely useful to refactor that code to make it changeable in one place in the codebase, so silverlight / mono / cf profiles could be added easily.

**Comment by flq** [comment]
Can you give a pointer as to where that code sits? I have found e.g. AssemblyReferenceExportBuilder.ProfileMatches which seems to be a mapping of compatible platforms, but I'm not sure...

**Comment by serialseb** [comment]
Yes, that's the place it currently is in. I'm not sure it still is a good idea to do this stuff though, as platform compat tables are unlikely to change very often.

**Comment by serialseb** [comment]
Reassigning to Future as not sure when we'll get to that.
